source/material_model/interface.cc

```diff
@@ -648,8 +634,43 @@ namespace aspect
         for (unsigned int i=0; i<values_out.additional_outputs.size(); ++i)
           values_out.additional_outputs[i]->average (operation, projection_matrix, expansion_matrix);
       }
+    }
 
+    namespace
+    {
+      std::vector<std::string> make_names()
```

Comment:
since the function is in a generic file (not seismic anisotropy-related), maybe call the function `make_seismic_additional_outputs_names()`?